FAQ.md

```diff
@@ -837,9 +837,18 @@ Count 就是暴力扫表，提高并发度能显著的提升速度，修改并
 - 测试大数据量的 count。
 - 调优 TiKV 配置，可以参考[性能调优](op-guide/tune-tikv.md)。
 
-#### 5.1.3 查看添加索引进度？
+#### 5.1.3 查看当前 DDL 的进度？
 
-通过 `admin show ddl` 查看当前添加索引 job。
+通过 `admin show ddl` 查看当前 job 进度。操作如下：
+```
```

Comment:
- Please leave a blank both before and after the code block.
- Add the code fencing "sql"?  "```sql"